apps/index.html

```diff
@@ -145,7 +145,7 @@ <h1>Sandstorm Apps</h1>
       </script>
 
       <div class="app">
-        <button data-app-id="h37dm17aa89yrd8zuqpdn36p6zntumtv08fjpu8a8zrte7q1cn60" data-package-id="3b04dd588ab7dbdf5a0299838fb3230d" data-package-url="http://sandstorm.io/apps/etherpad10.spk">Install »</button>
+        <button data-app-id="h37dm17aa89yrd8zuqpdn36p6zntumtv08fjpu8a8zrte7q1cn60" data-package-id="3b04dd588ab7dbdf5a0299838fb3230d" data-package-url="/apps/etherpad10.spk">Install »</button>
```

Comment:
Unfortunately, you got a bit ahead of yourself on these: The SPK files only exist on sandstorm.io, they aren't part of this repository, so never got deployed to sandstorm.org. We can keep the font source change above and the various blog links, but should still link to the .io SPK URLs for now, which appear to still work.

Reply:
Oh interesting... heh, this is one downside of local links. There are some local links to spks on this page. I'll fix those.

Reply:
Nevermind those were all my doing